serviceAccount/ServiceAccount.psm1

```diff
@@ -135,7 +135,7 @@ class AdfsRules
         {
             $rule = $this.rules[$i]
 
-            if ($rule.trim().Equals($ruleToFind.trim()))
+            if ($rule.Replace(' ','').trim() -eq $ruleToFind.Replace(' ','').trim())
```

Comment:
Are these rules case-insensitive? Might be a good idea to compare lower case

Reply:
The -eq comparison is case insensitive